### PR TITLE
[Client-gen] Move SwaggerSchema and ServerVersion to DiscoveryClient

### PR DESCRIPTION
**Pull request by caesarxuchao**
This PR move the SwaggerSchema method to the DiscoveryClient. This part of the effort of removing standalone method of [unversioned.Client](https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/client.go#L122).

First commit is #19298.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/19300)

### CODE REVIEW

**Comment by caesarxuchao** [comment]
@k8s-bot e2e test this please

File: pkg/client/unversioned/discovery_client.go [hunk 1]
```diff
@@ -46,6 +54,19 @@ type ServerResourcesInterface interface {
...
 }
 
+// ServerVersionInterface has a method for retrieving the server's version.
+type ServerVersionInterface interface {
+	// ServerVersion retrieves and parses the server's version (git version).
+	ServerVersion() (*version.Info, error)
+}
+
+// SwaggerSchemaInterface has a method to retrieve the swagger schema. Used in
+// client.Interface
```
Comment (nikhiljindal) [inline pkg/client/unversioned/discovery_client.go]: stale comment. Not used in client.Interface anymore

**Comment by nikhiljindal** [comment]
One nit, LGTM.
Feel free to self apply the LGTM label after fixing the comment.

**Comment by caesarxuchao** [comment]
@nikhiljindal, I addressed your both of your comments for #19298 and this PR. PTAL. Thanks.

**Comment by nikhiljindal** [comment]
LGTM.
Feel free to self apply the label once you have squashed

**Comment by caesarxuchao** [comment]
Apply the label per previous comment. Thanks @nikhiljindal!

**Pull request merged**